js/sliderView.js

```diff
@@ -1,363 +1,358 @@
-define([
-  'core/js/adapt',
-  'core/js/views/questionView',
-  'libraries/rangeslider'
-], function(Adapt, QuestionView) {
-
-  class SliderView extends QuestionView {
-
-    events() {
-      return {
-        'click .js-slider-number-click': 'onNumberSelected',
-        'focus .js-slider-item-input': 'onHandleFocus',
-        'blur .js-slider-item-input': 'onHandleBlur'
-      };
-    }
-
-    // Used by question to setup itself just before rendering
-    setupQuestion() {
-      if (this.model.get('_isSubmitted')) return;
-
-      this.selectItem(this.getIndexFromValue(this.model.get('_selectedItem').value));
-    }
+import Adapt from 'core/js/adapt';
+import QuestionView from 'core/js/views/questionView';
+import rangeslider from 'libraries/rangeslider';
+
+export default class SliderView extends QuestionView {
+
+  events() {
+    return {
+      'click .js-slider-number-click': 'onNumberSelected',
+      'focus .js-slider-item-input': 'onHandleFocus',
+      'blur .js-slider-item-input': 'onHandleBlur'
+    };
+  }
 
-    setupRangeslider() {
-      this.$sliderScaleMarker = this.$('.js-slider-number-selection');
-      this.$slider = this.$('.js-slider-item-input');
-      if (this.model.has('_scaleStep')) {
-        this.$slider.attr({ step: this.model.get('_scaleStep') });
-      }
+  // Used by question to setup itself just before rendering
+  setupQuestion() {
+    if (this.model.get('_isSubmitted')) return;
 
-      this.$slider.rangeslider({
-        polyfill: false,
-        onSlide: this.handleSlide.bind(this)
-      });
-      this.oldValue = 0;
+    this.selectItem(this.getIndexFromValue(this.model.get('_selectedItem').value));
+  }
 
-      if (this._deferEnable) {
-        this.setAllItemsEnabled();
-      }
+  setupRangeslider() {
+    this.rangeslider = rangeslider;
```

Comment:
weird addition, intentional?

Reply:
i get it, you had to declare and use the variable from the import?

Reply:
thats why I added it in yes